traitsui/tests/editors/test_file_editor.py

```diff
@@ -27,33 +26,95 @@ class FileModel(HasTraits):
 
     reload_event = Event()
 
+    existing_filepath = File(exists=True)
+
+
+@requires_toolkit([ToolkitName.qt, ToolkitName.wx])
+class TestSimpleFileEditor(BaseTestMixin, unittest.TestCase):
+    """ Test FileEditor (simple style). """
+
+    def setUp(self):
+        BaseTestMixin.setUp(self)
+
+    def tearDown(self):
+        BaseTestMixin.tearDown(self)
+
+    def test_simple_editor_set_text_to_nonexisting_path(self):
```

Comment:
From the Travis CI logs it looks like this test and `test_simple_editor_auto_set_text` are failing on wx
<details>

```
======================================================================
FAIL: test_simple_editor_auto_set_text (traitsui.tests.editors.test_file_editor.TestSimpleFileEditor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/.edm/envs/traitsui-test-3.6-wx/lib/python3.6/site-packages/traitsui/tests/editors/test_file_editor.py", line 78, in test_simple_editor_auto_set_text
    self.assertEqual(obj.filepath, "some_file.txt")
AssertionError: 'ome_file.txts' != 'some_file.txt'
- ome_file.txts
?             -
+ some_file.txt
? +
======================================================================
FAIL: test_simple_editor_set_text_to_nonexisting_path (traitsui.tests.editors.test_file_editor.TestSimpleFileEditor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/.edm/envs/traitsui-test-3.6-wx/lib/python3.6/site-packages/traitsui/tests/editors/test_file_editor.py", line 54, in test_simple_editor_set_text_to_nonexisting_path
    self.assertEqual(obj.filepath, "some_file.txt")
AssertionError: 'ome_file.txt' != 'some_file.txt'
- ome_file.txt
+ some_file.txt
? +
```

</details>
perhaps we skip them on wx and leave a comment?  They are allowed to fail, so CI is still green

Reply:
This is peculiar... they did pass on my host machine (OSX). Unfortunately I couldn't afford to dive deeper at the moment (sorry). I will add a comment and skip them here.

Reply:
No worries!